### DISCUSSION TITLE
Displaying local Administrator account name

### DISCUSSION THREAD

**Discussion by DaleMontgomery** [lithnet/access-manager#99] [Ideas]
I suggest adding the actual local administrator account name to the display once the request has been granted. 

We alter the local administrator account name by policy and thus it is not intuitively known. I understand this is not a value normally captured by Active Directory, but we are also using the Lithnet AM Agent in place of LAPS so I suspect you can alter the AD Schema with a new field and store the username there when the password is stored.

Even a short term solution like providing a place we can add a common comment for display on the password output page would be very helpful.

**Comment by ryannewington** [comment]
Hey @DaleMontgomery,

Thanks for the idea. We're actually working on a big update to the agent - and storing the username is already part of that update. 
The next iteration of the agent will actually store passwords in the AMS database, rather than AD, which allows us to open up scenarios for non-ad joined devices as well.

I'll provide an update here when we have something for you to test with.
